### PR TITLE
Fire blast

### PR DESCRIPTION
**Pull request by otaiwo70**

### CODE REVIEW

File: backend/game/cards/discard_cards/victory_point_manipulation_cards/fire_blast.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+from game.cards.discard_card import DiscardCard
+
+
+class FireBlast(DiscardCard):
+    def __init__(self):
+        Card.__init__(self, "Fire Blast", 3, "All other monsters - 2[health]", "")
+
+    def immediate_effect(self, player_that_bought_the_card, other_players):
+        for other_player in other_players:
+            other_player.update_energy_by(-2)
```
Comment (borisv13) [inline backend/game/cards/discard_cards/victory_point_manipulation_cards/fire_blast.py]: Why is energy being changed here? The card is about damaging other players.

File: backend/game/cards/discard_cards/victory_point_manipulation_cards/test_fire_blast.py [hunk 2]
```diff
@@ -0,0 +1,13 @@
+from game.cards.discard_cards.victory_point_manipulation_cards.fire_blast import FireBlast
+
+
+def test_fire_blast_subtracts_2_health(player, five_players):
+    for other_players in five_players:
+        other_players.update_health_by(12)
```
Comment (borisv13) [inline backend/game/cards/discard_cards/victory_point_manipulation_cards/test_fire_blast.py]: Why did we have to add 12 to all of the players? Why not just assert that their end health is starting health - 2?

File: backend/game/cards/discard_cards/victory_point_manipulation_cards/fire_blast.py (hunk 1 above)
Comment (otaiwo70) [reply backend/game/cards/discard_cards/victory_point_manipulation_cards/fire_blast.py]: Good catch, made the update to health and pushed in latest commit

File: backend/game/cards/discard_cards/health_manipulation_cards/test_fire_blast.py [hunk 3]
```diff
@@ -0,0 +1,13 @@
+from game.cards.discard_cards.health_manipulation_cards.fire_blast import FireBlast
+
+
+def test_fire_blast_subtracts_2_health(player, five_players):
+    for other_players in five_players:
+        other_players.current_health = 8
```
Comment (borisv13) [inline backend/game/cards/discard_cards/health_manipulation_cards/test_fire_blast.py]: Why is this for loop needed? Just assert result is 8 hp.

File: backend/game/cards/discard_cards/victory_point_manipulation_cards/test_fire_blast.py (hunk 2 above)
Comment (otaiwo70) [reply backend/game/cards/discard_cards/victory_point_manipulation_cards/test_fire_blast.py]: change made

File: backend/game/cards/discard_cards/health_manipulation_cards/test_fire_blast.py (hunk 3 above)
Comment (otaiwo70) [reply backend/game/cards/discard_cards/health_manipulation_cards/test_fire_blast.py]: for loop removed

File: backend/game/cards/discard_cards/health_manipulation_cards/test_fire_blast.py [hunk 4]
```diff
@@ -0,0 +1,11 @@
+from game.cards.discard_cards.health_manipulation_cards.fire_blast import FireBlast
+
+
+def test_fire_blast_subtracts_2_health(player, five_players):
+    FireBlast().immediate_effect(player, five_players)
+    assert all(other_players.current_health == 8
```
Comment (borisv13) [inline backend/game/cards/discard_cards/health_manipulation_cards/test_fire_blast.py]: DEFAULT_HEALTH - 2, not 8

**Pull request merged**